### DISCUSSION TITLE
Install Errors

### DISCUSSION THREAD

**Discussion by softelli** [GeekFunkLabs/fluidpatcher#54] [Q&A] [answered]
I can't install headless on raspberry pi model 3 B+.

Can anyone help me?

I'm trying this tutorial (https://www.youtube.com/watch?v=ONuXCzcuj7E)

The installation show me the next errors:

[...]
Installing/Updating FluidPatcher version 0.7.9 ...
find: Must separate multiple arguments to -type using: ','
cp: não foi possível criar arquivo comum '.././utils/__init__.py': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././utils/README.md': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././utils/stompboxpi.py': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././patcher/fpyaml.py': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././patcher/__init__.py': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././patcher/README.md': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././patcher/file_formats.md': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././patcher/fswrap.py': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././assets/gfl_logo.ico': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././assets/patchcord.c': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././assets/fplessons.png': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././assets/squishbox-install.bash': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././SquishBox/midi/elevatorgroove.mid': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././.github/ISSUE_TEMPLATE/help-request.md': Arquivo ou diretório inexistente
cp: não foi possível criar arquivo comum '.././utils/hw_overlay.py': Arquivo ou diretório inexistente
gcc: error: assets/patchcord.c: Arquivo ou diretório inexistente
gcc: fatal error: no input files
compilation terminated.
mv: não foi possível obter estado de 'patchcord.so': Arquivo ou diretório inexistente
Compiling latest FluidSynth from source...
Getting build dependencies...

[ ... and ...]

Setting up web-based file manager...
Updating apt indexes...
server {
        listen 80 default_server;
        listen [::]:80 default_server;
        root /var/www/html;
        index index.php index.html index.htm index.nginx-debian.html;
        server_name _;
        location / {
                try_files $uri $uri/ =404;
        }
        location ~ \.php$ {
                include snippets/fastcgi-php.conf;
                fastcgi_pass unix:/run/php/php-fpm.sock;
        }
}
sed: não foi possível ler /etc/php//fpm/php.ini: Arquivo ou diretório inexistente
sed: não foi possível ler /etc/php//fpm/php.ini: Arquivo ou diretório inexistente
sed: não foi possível ler /lib/systemd/system/php-fpm.service: Arquivo ou diretório inexistente
sed: não foi possível ler /lib/systemd/system/php-fpm.service: Arquivo ou diretório inexistente

**Comment by albedozero** [comment, marked as answer]
Thanks for the catch - there was a typo in the last update that should be fixed now.

**Reply by tizfra** [reply]
Hi, I'm not able to update squishbox to 0.7.11 version from 0.7.9 from software option.

**Reply by albedozero** [reply]
Thanks - this is fixed if you re-run the update script. Check out my comments in #55
